apps/GNOME/uninstall

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || error "Could not remove GNOME and its software."
```

Comment:
still wrong
purge_packages does not take any input
you just call purge_packages || exit 1
that is it